blueprints/component/index.js

```diff
@@ -62,8 +62,8 @@ module.exports = {
         templatePath   = pathUtil.getRelativeParentPath(options.entity.name) +
           'templates/components/' + stringUtil.dasherize(options.entity.name);
       }
-      importTemplate   = 'import layout from \'' + templatePath + '\';\n';
-      contents         = '\n  layout';
+      importTemplate   = '// @ts-ignore: Ignore import of compiled template\nimport layout from \'' + templatePath + '\';\n';
+      contents         = '\n  layout = layout';
```

Comment:
[nit] Can we add a semicolon after `= layout`, since the blueprints tend not to rely on ASI?

Reply:
oops!